docs/source/posts/2022/2023-02-01-final-report-shivam.rst

```diff
@@ -0,0 +1,406 @@
+   physical properties of the object surface into account. There are
+   different representations of these physical material properties. One
+   that is frequently used is the metallic-roughness-model. We have
+   various material properties already in FURY, we need to apply it to
+   glTF models as well.
+
+
+Skinning for models with no indices
+***********************************
+
+   The glTF format supports non-indexed geometry (e.g., the ``Fox``
+   model). We currently do not know how to render the model without
+   indices. I tried estimating it in this
+   `branch <https://github.com/xtanion/fury/blob/gltf-indices-fix/fury/gltf.py>`__.
+   However, It fails to render in skinning.
+
+   *Branch URL:*
+
+   -  **Rendering glTF with no indices: (in-progress)**
+      https://github.com/xtanion/fury/blob/gltf-indices-fix/fury/gltf.py
+
+Other Objectives
+----------------
+
+
+Fetcher for importing glTF files from Khronos-glTF-Samples
+**********************************************************
+
+   The
+   `KhronosGroup/gltf-samples <https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/>`__
+   contain multiple glTF sample models to test a glTF viewer for free.
+   Implemented new methods in fetcher that can load all of these models
+   by (usind type) asynchronously. The glTF fetcher is capable
+   of the following:
+
+   -  Downloading multiple models asynchronously.
+   -  Get the path to the downloaded model using it   -  Download any model using the URL of the model.
+
+   *Pull Requests:*
+
+   -  **Fetching glTF sample models from github: (Merged)**
+      https://github.com/fury-gl/fury/pull/602
+   -  **Fixing github API limit: (Merged)**
+      https://github.com/fury-gl/fury/pull/616
+
+
+Other Pull Requests
+*******************
+
+   -  **Sphere actor uses repeat_primitive by default**:
+      `fury-gl/fury/#533 <https://github.com/fury-gl/fury/pull/533>`__
+   -  **Cone actor uses repeat primitive by default**:
+      `fury-gl/fury/#547 <https://github.com/fury-gl/fury/pull/547>`__
+   -  **Updated code of viz_network_animated to use fury.utils**:
+      `fury-gl/fury/#556 <https://github.com/fury-gl/fury/pull/556>`__
+   -  **Added simulation for Tesseract**:
+      `fury-gl/fury/#559 <https://github.com/fury-gl/fury/pull/559>`__
+   -  **GLTF actor colors from material**
+      `fury-gl/fury/#689 <https://github.com/fury-gl/fury/pull/689>`__
+
+
+GSoC weekly blogs
+*****************
+
+   -  My blog posts can be found on the `FURY
+      website <https://fury.gl/latest/blog/author/Shivam-Anand.html>`__
+      and the `Python GSoC
+      blog <https://blogs.python-gsoc.org/en/xtanions-blog/>`__.
+
+Timeline
+--------
+
+
++---------------------------+-----------------+------------------------+
+| Date                      | Description     | Blog Link              |
++===========================+=================+========================+
+| Week 0(24-05-2022)        | My Journey to   | `F                     |
```

Comment:
blog link does not work, to update:

![image](https://user-images.githubusercontent.com/23106443/216669393-4fda922e-6039-4bc5-b778-a94326c91f6c.png)